config/vsphere-csi-driver/patches/daemonset-csi-node-driver-registrar-initcontainer.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: vsphere-csi-node
+  namespace: vmware-system-csi
+spec:
+  template:
+    spec:
+      initContainers:
+        - name: wait-for-crd
+          image: gsoci.azurecr.io/{{ .Values.initContainer.image.name }}:{{ .Values.initContainer.image.tag }}
+          command:
+            - /bin/bash
+            - -c
+            - |
+              echo "Waiting for CRD csinodetopologies.cns.vmware.com to be registered..."
+              attempt=0
+              max_attempts=30
+              while ! kubectl get crd csinodetopologies.cns.vmware.com >/dev/null 2>&1; do
```

Comment:
```suggestion
              while ! kubectl get crd csinodetopologies.cns.vmware.com &> /dev/null; do
```

Reply:
sure, but this is bash-specific - i try not to get too attached to things which don't port to other shells (especially when writing shell scripts)